cache/lru_cache.cc

```diff
@@ -407,11 +407,10 @@ void LRUCacheShard::Promote(LRUHandle* e) {
     }
   } else {
     // Since the secondary cache lookup failed, mark the item as not in cache
-    // and charge the cache only for metadata usage, i.e handle, key etc
+    // Don't charge the cache as its only metadata that'll shortly be released
```

Comment:
Is this implementation specific ? Maybe other secondary cache implementation will not allocate a handle and charge the cache for metadata usage when it fails. This can be a short term fix. Maybe later on, we can make some change to the secondary interface and at least the secondary cache can tell RocksDB if it charged or not.